extra/src/main/java/aima/extra/probability/proposition/Proposition.java

```diff
@@ -0,0 +1,43 @@
+package aima.extra.probability.proposition;
+
```

Comment:
why are the proposition classes still around, I thought we were getting rid of these in preference to using predicates?